style.css

```diff
@@ -0,0 +1,332 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+

```

Comment:
- [ ] Kindly make sure your page fits the entire screen size, The user doesn't need to scroll to the right to see the whole page.
`Kindly you may consider giving you page of 100%`

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205616766-20bdc618-0260-4bd7-bb47-c988ce09925c.png)
![image](https://user-images.githubusercontent.com/97100091/205616871-d38abc71-52ea-479c-8c8b-28a7a97bfcb2.png)


Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205617230-f1e03952-c89d-43a2-8d79-67fcd76aae67.png)